### ISSUE TITLE
Waypoints not working

### ISSUE THREAD

**Issue by LeanieBean** [FreeCodeCamp/FreeCodeCamp#3275]
I have tried the first 3 Waypoints and am experiencing 3 problems: 1 - no step-by-step checks below the 'Run test' and other buttons; 2 - mobile screen is completely blank; 3 - 'Run test' isn't working. Nothing happens. Here's my screengrab:

![waypoint screenshot](https://cloud.githubusercontent.com/assets/13335415/9833632/c7a52808-5995-11e5-9fb9-ae2440c562b7.jpg)

Many thanks!

**Comment by izybit** [comment]
Can you test if you have the same problems with firefox or chrome?

**Comment by catapixel** [comment]
@LeanieBean I would try clearing your browser's cache of temporary files. All of your code is stored locally and a previous problem the site encountered, possibly with another code submission, is probably causing your issues. Also, on this one, and I believe other waypoints as well, nothing gets processed/checked until you click the "Run tests" button.

**Comment by LeanieBean** [comment]
@catapixel I have cleared out the cache and am still having problems. Also a new one ... it is morning here and so I have just signed in but when I come to run my first test there is a yellow button below the others saying 'sign in', so I sign in again but the yellow button is still there. And no checklist below the buttons and still no text in the mobile screen on the right.

@izybit Everything working fine in Chrome so will carry on there.
